docs/opcodes/60.mdx

```diff
@@ -3,19 +3,25 @@ fork: Frontier
 group: Push Operations
 ---
 
+*Index 1 is top of the stack.*
+
+## Notes
+
+The new value is put on top of the stack, incrementing all the other value indices. The values for a specific opcode thus have to be pushed in reverse order of the stack. For example, with [MSTORE](/#52), the first value pushed would have to be `value`, and then `offset`.
+
 ## Stack output
 
 0. `value`: pushed value, aligned to the right (put in the lowest significant bytes).
 
 ## Example
 
 | Code |
-|------|
+|-----:|
 | `0x60FF` |
 
-| Input | Output |
-|------:|-------:|
-| | `0xFF` |
```

Comment:
nit: maybe we could add a 2nd push here to visualize that it's a LIFO (the notes are also sufficient by themselves)

Reply:
good idea, done